### ISSUE TITLE
the problem is the addition of the Bashkir language

### ISSUE THREAD

**Issue by tc-dl** [espeak-ng/espeak-ng#575]
in the process of creating the Bashkir language a problem arose. when playing a voice, writes error: the specified espeak-ng voice does not exist.
adhering to the instruction: http://jrmeyer.github.io/tts/2016/07/03/How-to-Add-a-Language-to-eSpeak-NG.html created duplicate files of the Tatar voice since these languages ​​are similar.
as I did: in espeak-ng data / lang / trk I created ba with the text name Bashkir language ba. In the dictsource folder, the Tatar dictionaries were copied under the name ba_extra, ba_list, ba_rules. in the folder phsource also copied the file ph_bashkir, in the file phonemes added: phonemetable ba base1 include ph_bashkir. in the root folder in Makefile.am made the following changes: espeak-ng-data / ba_dict \ ba: espeak-ng-data / ba_dict espeak-ng-data / ba_dict: dictsource / ba_list dictsource / ba_rules dictsource / ba_extra. when make and sudo make install are executed, the ba_dict file is successfully compiled in the espeak-ng-data folder. but there is still no voice.
after a thorough examination of the code, I did not find anything. tell me what I did wrong? Or something did not finish?

**Comment by tc-dl** [comment]
https://github.com/tc-dl/espeak-ba-files.git

**Comment by tc-dl** [comment]
I certainly have doubts in the makefile.am

**Comment by valdisvi** [comment]
Review your procedures according to the project's guide [Adding or Improving a Language](https://github.com/espeak-ng/espeak-ng/blob/master/docs/add_language.md).

**Comment by tc-dl** [comment]
I do everything according to the instructions all of the appropriate libraries documentation set. makefile compiles dikt file, but after I enter the command (sudo make install) passes with no error, when checking voice paplay persistently displays: error: the specified espeak-ng voice does not exist.

**Comment by valdisvi** [comment]
When I tested this, I got strange problem with completely different language (Greenlandic — kl), could not resolve it, and left it for later.
When @rhdunn caught problems by adding more voices and fixed it in f58f665336772e9bc161a19442074c76d2b5f8ba, it become clear, problem with this was also related to number of languages reaching predefined limit.
It is included now in espeak-ng project: f4c4cd32082c757ea035497b816321d0d64e9826.
Thanks for contribution!
